developer-extension/src/background/store.ts

```diff
@@ -7,16 +7,32 @@ export const store: Store = {
   useDevBundles: false,
   useRumSlim: false,
   blockIntakeRequests: false,
+  local: {},
 }
 
 export function setStore(newStore: Partial<Store>) {
-  if (wouldModifyStore(newStore)) {
+  if (wouldModifyStore(newStore, store)) {
     Object.assign(store, newStore)
     sendAction('newStore', store)
     chrome.storage.local.set({ store })
   }
 }
 
+export function setLocalStore(newStore: Partial<LocalStore>, tabId: number) {
+  if (!store.local[tabId]) {
+    store.local[tabId] = {
+      rumConfig: {},
+      logsConfig: {},
+    }
+  }
+  const localStore = store.local[tabId]
+  if (wouldModifyStore(newStore, localStore)) {
+    Object.assign(localStore, newStore)
+    sendAction('newStore', store)
+    chrome.storage.local.set({ store })
```

Comment:
I'm not that sure we should persist "local store", because tabId are likely to be changed/reused between launches. I think we should keep it in memory.

Reply:
For now, it is not an issue since even if it was stored in memory, the content stored in the tab is refreshed regularly.
Any reason to make the change now or could it wait for a future usage?

Reply:
Ok, makes sense, it can be done later :)